src/guide/receive_payment.md

```diff
@@ -321,3 +316,30 @@ When waiting for a payment to be received, it is enough to wait for a <code>Paym
 ```
 </section>
 </custom-tabs>
+
+## Event Flows
+Once a receive payment is initiated, you can follow and react to the different payment events using the guide below for each payment method. See [Listening to events](/guide/events.html) for how to subscribe to events.
+
+### Lightning
+| Event | Description | UX Suggestion |
+| --- | --- | --- |
+| **PaymentPending** | The swap service has intercepted a HTLC for the Lightning invoice and broadcast a lockup transaction. The SDK has seen the lockup transaction and will broadcast the claim transaction, either when the lockup transaction is confirmed or if it is accepted as a zero-conf payment. | Show payment as pending. |
```

Comment:
Perhaps no need to zoom in all the way to the HTLC? It actually may receive several HTLCs and will only send the lockup when enough funds are held. Maybe something more higher level such as "The swap service is holding an incoming payment" or something similar?